amethyst_input/src/button.rs

```diff
@@ -10,7 +10,7 @@ pub enum Button {
     Key(VirtualKeyCode),
 
     /// Scan code from keyboard, use this when the position of the key matters
```

Comment:
What does position of the key mean, exactly?

Reply:
"geographical" location of the key (er, not sure a better word for this).
Like, on US keyboard layouts, there's `qwerty`. Some other keyboards might be `azerty`, but you want the top left key